ethereum/spec/src/main/java/tech/pegasys/teku/spec/datastructures/epbs/versions/gloas/ExecutionPayloadBid.java

```diff
@@ -11,9 +11,10 @@
  * specific language governing permissions and limitations under the License.
  */
 
-package tech.pegasys.teku.spec.datastructures.execution.versions.gloas;
+package tech.pegasys.teku.spec.datastructures.epbs.versions.gloas;
```

Comment:
why epbs package here?

Reply:
it's a new package we introduced for all epbs objects, we could move them around in the future.

Reply:
If it is temporary, ok:)